ledger-api/grpc-definitions/com/daml/ledger/api/v1/event.proto

```diff
@@ -54,9 +56,22 @@ message CreatedEvent {
   Value contract_key = 7;
 
   // The arguments that have been used to create the contract.
-  // Required
+  // Set if there was a match by template_id in the given transaction filter.
+  // Optional
   Record create_arguments = 4;
 
+  // Opaque representation of contract payload intented for forwarding
+  // to an API server as a contract disclosed as part of a command
+  // submission.
+  // Optional
+  google.protobuf.Any create_arguments_blob = 10;
```

Comment:
This assumes that the `create_arguments_blob` is a protobuf message albeit of a type unknown at this spot. Should this rather be a `bytes` field?

Reply:
See https://github.com/digital-asset/daml/pull/13912#discussion_r885557039